src/main/java/org/enginehub/discord/module/IdleRPG.java

```diff
@@ -231,6 +231,10 @@ public void onInitialise() {
         } catch (IOException e) {
             e.printStackTrace();
         }
+
+        // Purge players who only got to level 1 over a week ago.
+        players.values().removeIf(data -> data.level == 1
+            && data.levelTime.isBefore(Instant.now().minus(1, ChronoUnit.WEEKS)));
```

Comment:
Maybe worth moving this check to the `data` class and doing `players.values().removeIf(PlayerData::isNotParticipating)`.